static/css/foundation.css

```diff
@@ -1,4987 +0,0 @@
-@charset "UTF-8";
-/**
- * Foundation for Sites by ZURB
- * Version 6.4.2
```

Comment:
You're upgrading Foundation 6.4.2 to 6.7.3. Have you checked the changes and verified there'll be no changes that break other things?

Reply:
Yes, I checked all pages to make this sure and didn't find anything breaking.